### PR TITLE
add JuliaLang

### PR DESCRIPTION
**Pull request by marinlauber**
Julia examples for the `loop` and `fibonacci` benchmark in JuliaLang.

The simplest way to run these is to use `juliaup` to download the latest Julia release; see https://github.com/JuliaLang/juliaup.
ad then simply with
```bash
julia code.jl ARGS
```

### CODE REVIEW

File: loops/julia/code.jl [hunk 1]
```diff
@@ -0,0 +1,12 @@
+let
+    a = zeros(Int,10^4)
+    u = parse(Int,ARGS[1])
+    r = only(rand(1:10^4,1))
+    @inbounds for i ∈ 1:10^4
+        @inbounds for j ∈ 1:10^5
+            a[i] = a[i] + u
```
Comment (silva96) [inline loops/julia/code.jl:7]: You are not doing the module operation here, see the other languages.

**Review by silva96 — CHANGES_REQUESTED** [review]

File: loops/julia/code.jl (hunk 1 above)
Comment (marinlauber) [reply loops/julia/code.jl]: You are right, I missed that!

File: loops/julia/code.jl [hunk 2]
```diff
@@ -0,0 +1,12 @@
+let
+    a = zeros(Int,10^4)
+    u = parse(Int,ARGS[1])
+    r = rand(1:10^4)
+    @inbounds for i ∈ 1:10^4
+        @inbounds for j ∈ 1:10^5
```
Comment (serhii-havrylov) [inline loops/julia/code.jl:6]: it should be `j ∈ 0:(10^5-1)` to correspond to other languages

File: fibonacci/julia/code.jl
[the remark this answers is not in the document]
Comment (bddicken) [reply fibonacci/julia/code.jl:1]: Yeah I'm pretty sure this is "cheating". We need to make it an actual function to be apples to apples. I'm gonna go ahead and merge and we can fix later.

**Comment by PEZ** [comment]
I'd love for Julia to join the list of languages that are represented in the new in-process benchmarks.

* #371
* #410

Julia was one of my reasons to insist on shifting to benchmarking without start-times involved.

**Pull request merged**